packages/fields/src/Implementation.js

```diff
@@ -29,15 +30,19 @@ class Field {
     // Should be overwritten by types that implement a Relationship interface
     this.isRelationship = false;
 
-    this.access = parseFieldAccess({
+    this.access = this.parseFieldAccess({
       schemaNames,
       listKey,
       fieldKey: path,
       defaultAccess,
-      access: access,
+      access,
     });
   }
 
+  parseFieldAccess(args) {
```

Comment:
If we're going to make this change in this PR we should make sure we reference it in the changelog as a `minor`.